pkg/controllers/common/services.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import "github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+
+type IVPCService interface {
+	RegisterNamespaceNetworkconfigBinding(ns string, ncCRName string)
```

Comment:
Please Add some description to tell users what's the purpose of this interface function.